.eslintrc.js

```diff
@@ -175,6 +175,12 @@ module.exports = {
             'error',
             { vars: 'all', args: 'none', ignoreRestSiblings: true, varsIgnorePattern: '^_' },
         ],
+        '@typescript-eslint/no-restricted-imports': [
+            'error',
+            {
+                paths: [{ name: './' }, { name: '../.' }, { name: '../../.' }],
```

Comment:
please change this to `.`, `..`, `../..` lash dot or slash is being removed by prettier, and please verify it will really throw if you use that import

Reply:
fixed